OTKit/otkit-colors/colors.yml

```diff
@@ -1,3 +1,49 @@
 props:
-  # TODO
-  
\ No newline at end of file
+
+  # Primary Action
+  # This is only used on objects, backgrounds with large text (24px or larger, bold), and will be used as a color to guide users towards actionable content.
+  # =============================================
+  
+  color-primary:
+    value: "#F91F14"
+  
+  color-primary-active:
+    value: "#FB665F"
+  
+  color-primary-disabled:
+    value: "rgba(#F91F14, 0.40)"
+  
+  # Secondary Action
+  # This color is only to be used on text sizes below 24px for actionable content (tabs, links, underlines, points)
+  # =============================================
+  
+  color-secondary:
+    value: "#E81106"
+  
+  color-secondary-active:
+    value: "#FA4940"
+  
+  color-secondary-disabled:
+    value: "rgba(#E81106, 0.40)"
+  
+  # Primary Gray, used on all text sizes 
+  # =============================================
+  
+  color-primary-gray:
```

Comment:
I think  the gray variable names should be color-gray-primary, color-gray-secondary, color-gray-utility.

This is because color-primary-gray sounds like a subset of color-primary, which it isn't. Also, when you name it color-gray-[x], it's easier to autocomplete on a text editor when you want to include a gray colored text.